### DISCUSSION TITLE
Banned from Discord server for unknown reason

### DISCUSSION THREAD

**Discussion by kevindd992002** [sct/overseerr#3855] [Help]
Hello devs/mods. I'm not sure why but I was recently banned from your Discord server. What does it take for me to get reinstated? Can you please let me know what my violation was? I tried reaching out in Reddit but I haven't had luck so I'm trying here. Thanks.

**Comment by nmbrg** [comment]
i too was banned a long time ago, still have no clue on why. i have little to no hope on getting back either seeing theres no way to contact them about that. and the response here from the devs are minimum when ive created discussions/issues on other stuff.

sucks when issues arise that needs help from the community. but not much to do about it so i guess its just to move on in life. :)

**Comment by kevindd992002** [comment]
Well, to be fair, they responded to me in reddit a few days ago. Hopefully, they follow through though.

**Reply by nmbrg** [reply]
Good for you. :) Hope it will resolve.

**Comment by kevindd992002** [comment]
Until now they never got back to me with a solution. Sigh!

**Reply by nmbrg** [reply]
Well, seeing as they haven’t updated overseerr for close to a year now it looks more and more like abandonware at the moment.

(someone called it “doneware” but that has to be the joke of the century.)

so i doubt the discord are any livelier.. granted it feels bad beeing banned for something you can’t even remember the reason for (if there ever were a reason), I doubt we’re missing out at the moment.

**Reply by kevindd992002** [reply]
> Well, seeing as they haven’t updated overseerr for close to a year now it looks more and more like abandonware at the moment.
> 
> (someone called it “doneware” but that has to be the joke of the century.)
> 
> so i doubt the discord are any livelier.. granted it feels bad beeing banned for something you can’t even remember the reason for (if there ever were a reason), I doubt we’re missing out at the moment.

Ohh! I didn't know it wasn't updated for close to a year now. That is a bummer indeed. That explains why some feature request aren't being implemented. There must be a reason why they are abandoning this because it's a really useful software.
